### DISCUSSION TITLE
Is there a way to set the radius / connection of each point in heatmap?

### DISCUSSION THREAD

**Discussion by endolith** [python-visualization/folium#1974] [Q&A]
ChatGPT introduced me to folium and I like that it shows heatmaps overlaid on map tiles, but the heatmap is made up of discrete points that blend together when zoomed out:

![image](https://github.com/python-visualization/folium/assets/58611/9cabed62-d97b-49d2-b3ad-f5b02a897f9e)

![image](https://github.com/python-visualization/folium/assets/58611/604217f9-fec2-4ecc-84f4-01167968695b)

1. Is there a way to make the points vary in size depending on HDOP data, so they behave like a kernel density estimation of true position when the GPS device is stationary?

It looks like the `radius` parameter is an int, not an array?

```py
    radius : int, default 25
        Radius of each "point" of the heatmap
```

2. Is there a way to connect the points in the order they were recorded, so that the heatmap is interpolated between each neighboring point?  For a GPS track we know the device was traveling between the points, and it didn't just magically disappear from one point and appear at another.

**Comment by hansthen** [comment]
Hi endolith,

Sorry for the late response. Folium is a thin wrapper around the Leaflet javascript library. For these types of questions you'd better contact the upstream plugin maintainer. 

Regards,

Hans Then
